jsapp/scss/components/_kobo.map.scss

```diff
@@ -166,6 +167,49 @@ body .leaflet-control-layers,
 
   .map-marker-item {
     padding: 4px 6px;
+    position: relative;
+
+    &:after {
+      position: absolute;
+      display: inline-block;
+      font-family: 'FontAwesome';
+      font-size: 10px;
+      right: 6px;
+      top: 8px;
+      opacity: 0.5;
+    }
+
+    &.selected {
+      background: #F5F5F5;
+
+      &:after {
+        content: "\f111";
+      }
+    }
+
+    &.unselected {
+      opacity: 0.6;
+
+      &:after {
+        content: "\f10c";
```

Comment:
Maybe add a small comment here and in previous one about which FontAwesome shape you're using here?